src/main/java/edu/unc/lib/boxc/migration/cdm/services/AspaceRefIdService.java

```diff
@@ -55,10 +54,18 @@ public AspaceRefIdService() {
     public void generateBlankAspaceRefIdMapping() throws IOException {
         assertProjectStateValid();
 
+        if (!hasProjectContriAndDescriFields()) {
```

Comment:
hmm, i don't know if this is desirable, you might want to check in with anna. This would mean we wouldn't be able to generate blank mappings for non-cdm collections without adding in vestigial fields to the project. It would probably be fine to have an empt hookid column in the csv file. 

I know it's awkward dealing with fields that may or may not be there in sql tables. A more complicated option would be to have it run a different getIdsAndHookIds query that doesn't retrieve to the hookid field if `hasProjectContriAndDescriFields` returns false (it'd just return "" in the select clause), but I don't know if that's essential.